### PR TITLE
docs: update `stats/iter` examples

### LINKED ISSUE #1652: [RFC]: improve README examples of `stats/iter` namespace
### Description

This RFC proposes to improve the [README examples](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/iter#examples) of the `stats/iter` namespace package. Currently, namespace package READMEs only include minimal examples which are not particularly informative and do not showcase namespace functionality.

Historically, namespace examples were not given much effort due to the rapidly changing nature of the project; however, now that development and organization has settled, it would be good to revisit these packages and add proper examples to better communicate to users how to use namespace contents.

For those wanting to contribute to this issue, you can use the following steps:

- Browse through the [contents](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/iter) of the `stats/iter` namespace and identify interesting functions and packages to showcase.
- Update the example code block in the "Examples" section of the README for the `stats/iter` namespace and open a PR.
- When updating the README, examples should follow existing project conventions (e.g., ES5).
- Examples may `require` functionality from other packages to help make examples more interesting.

### Related Issues

None.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by ptl-harsh**
Fixes #1652 
Resolves #1652  .

## Description
Added few examples showcasing the usage of functions from the @stdlib/stats/iter namespace

## Related Issues

> Does this pull request have any related issues?
No

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/stats/iter/README.md [hunk 1]
```diff
@@ -128,91 +174,91 @@ console.log( getKeys( ns ) );
 
 <!-- <toc-links> -->
 
-[@stdlib/stats/iter/cugmean]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/iter/cugmean
+[@stdlib/stats/iter/cugmean]: https://github.com/stdlib-js/stats/tree/main/iter/cugmean
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/iter/README.md]: Why did you change these URLs? These changes need to be reverted. All URLs within the main project repo should refer to resources in the main project repo, not standalone repos.

File: lib/node_modules/@stdlib/stats/iter/README.md [hunk 2]
```diff
@@ -103,11 +103,57 @@ var o = ns;
...
+
 ```javascript
-var getKeys = require( '@stdlib/utils/keys' );
-var ns = require( '@stdlib/stats/iter' );
 
-console.log( getKeys( ns ) );
+// Computing Cumulative Sum
+
+
+var ns = require('@stdlib/stats/iter');
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/iter/README.md:110]: Please follow stdlib style and format conventions.

**Review by kgryte — CHANGES_REQUESTED** [review]
Thanks, @ptl-harsh, for opening this PR. I've left an initial set of review comments.

File: lib/node_modules/@stdlib/stats/iter/README.md (hunk 2 above)
Comment (ptl-harsh) [reply lib/node_modules/@stdlib/stats/iter/README.md:110]: May u please help me in figuring out the correct style and format conventions

Comment (kgryte) [reply lib/node_modules/@stdlib/stats/iter/README.md:110]: I suggest studying the existing codebase and attempting to mimic what you see. E.g., spacing, quote style, etc.

**Review by ptl-harsh — COMMENTED** [review]
Updated as per the standard style and format conventions

**Review by ptl-harsh — COMMENTED** [review]
Updated as per the standard style and format conventions
